### PR TITLE
[zh] Resync container-runtimes page

### PR DESCRIPTION
**Pull request by tengqm**
closes: #28469

### CODE REVIEW

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -199,7 +202,8 @@ Install containerd:
 {{% tab name="Windows (PowerShell)" %}}
 
 <!--
-Start a Powershell session, set `$Version` to the desired version (ex: `$Version=1.4.3`), and then run the following commands:
+Start a Powershell session, set `$Version` to the desired version (ex: `$Version=1.4.3`),
+and then run the following commands:
 -->
 启动 Powershell 会话，将 `$Version` 设置为所需的版本（例如：`$ Version=1.4.3`），
```
Comment (chenxuc) [inline content/zh/docs/setup/production-environment/container-runtimes.md:208]: It's not related to the issue of the PR. But there's one extra space in `$ Version=1.4.3` which makes it an invalid command.

Comment (tengqm) [reply content/zh/docs/setup/production-environment/container-runtimes.md]: Good catch. Fixed.

**Review by chenrui333 — APPROVED** [review]
/approve

**Pull request merged**